src/modules/toolbox/components/shareButton/ShareDialogContent.js

```diff
@@ -129,8 +150,8 @@ export class ShareDialogContent extends MvuElement {
 	}
 
 	set shareurls(value) {
```

Comment:
Let's add simple documentation about `value`. e.g. value could either be a plain string or an object containing the keys `fileId` and `adminId`